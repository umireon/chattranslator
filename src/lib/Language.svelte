<script lang="ts" context="module">
  export const defaultTargetLanguageCode = 'en'
</script>

<script lang="ts">
  export let targetLanguageCode: string
</script>

<main>
  <p>
    <label for="targetLanguage">en</label>
    <input id="targetLanguage" type="text" bind:value={targetLanguageCode} />
  </p>
</main>
