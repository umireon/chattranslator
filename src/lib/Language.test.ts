import Language from './Language.svelte'
import { render } from '@testing-library/svelte'

test('Language snapshot', () => {
  const props = {
    targetLanguageCode: 'en',
  }
  const component = render(Language, { props })
  expect(component.container).toMatchSnapshot()
})
