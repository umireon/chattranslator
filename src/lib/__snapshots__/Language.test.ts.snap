// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Language snapshot 1`] = `
<body>
  <div>
    <main>
      <p>
        <label
          for="targetLanguage"
        >
          Target Language
        </label>
         
        <input
          id="targetLanguage"
          list="targetLanguageList"
          type="text"
        />
         
        <datalist
          id="targetLanguageList"
        >
          <option
            value="de"
          />
          <option
            value="en"
          />
          <option
            value="es"
          />
          <option
            value="fr"
          />
          <option
            value="it"
          />
          <option
            value="ja"
          />
          <option
            value="ko"
          />
          <option
            value="pl"
          />
          <option
            value="pt"
          />
          <option
            value="ru"
          />
          <option
            value="th"
          />
          <option
            value="tr"
          />
          <option
            value="zh"
          />
        </datalist>
      </p>
    </main>
  </div>
</body>
`;
